src/database.cpp

```diff
@@ -36,6 +36,8 @@ void Database::CheckEvent(Event*& event) const {
 			return ;
 	}
 	this->AfterCheck(*event);
+	if (event->HasErrorOccurred())
+			return ;
```

Comment:
一番最後に確認しても意味ありません。
AfterCheck関数の前でエラー確認を行ってください。
```suggestion
	if (event->HasErrorOccurred())
			return ;
	this->AfterCheck(*event);
```

Reply:
了解です、再度修正し、プシューしました。